modules/openapi-generator/src/test/java/org/openapitools/codegen/java/spring/SpringCodegenTest.java

```diff
@@ -1696,4 +1696,30 @@ public void shouldNotUseEqualsNullableForArrayWhenNotSetInConfig_issue13385() th
                 .assertMethod("equals")
                 .bodyContainsLines("return Arrays.equals(this.picture, testObject.picture);");
     }
+
+    @Test
+    public void testEnumUnknownDefaultCaseDeserializationTrue_issue13241() throws IOException {
+
+        SpringCodegen codegen = new SpringCodegen();
+        codegen.setLibrary(SPRING_BOOT);
+        codegen.additionalProperties().put(CodegenConstants.ENUM_UNKNOWN_DEFAULT_CASE, "true");
+
+        Map<String, File> files = generateFiles(codegen, "src/test/resources/bugs/issue_13241.yaml");
+
+        JavaFileAssert.assertThat(files.get("Color.java")).printFileContent()
+                .assertMethod("fromValue").bodyContainsLines("UNKNOWN_DEFAULT_OPEN_API");
```

Comment:
Could you please add `return` to body contains check? Like this
`.bodyContainsLines("return UNKNOWN_DEFAULT_OPEN_API;");`

Also please remove `printFileContent()`. That's for local test purpose only

Reply:
@borsch I have updated the tests with the requested changes. Thanks